packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -917,22 +909,21 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
         _transformationController!.value,
         focalPointSceneScaled - focalPointScene,
       );
-      if (widget.onInteractionStart != null) {
-        widget.onInteractionStart!(
-            ScaleStartDetails(focalPoint: focalPointSceneScaled)
-        );
-      }
-      if (widget.onInteractionUpdate != null) {
-        widget.onInteractionUpdate!(ScaleUpdateDetails(
-          rotation: 0.0,
-          scale: scaleChange,
-          horizontalScale: 1.0,
-          verticalScale: 1.0,
-        ));
-      }
-      if (widget.onInteractionEnd != null) {
-        widget.onInteractionEnd!(ScaleEndDetails());
-      }
+      widget.onInteractionStart?.call(
```

Comment:
If the developer tries to access the transform in the `onInteractionStart`, are they supposed to see the updated transform, or the pre-scroll transform?

Reply:
Good point, I updated the docs.  For start, the transformation shouldn't have changed, but for update/end, it should have changed.